image/golem_blender_app/golem_blender_app/verifier_tools/image_metrics_calculator.py

```diff
@@ -129,15 +129,28 @@ def get_file_extension_lowercase(file_path):
     return os.path.splitext(file_path)[1][1:].lower()
 
 
+def check_exr_multilayer(image_path):
+    # Might exit()
+    image_path = Path(image_path)
+    try:
+        channels = OpenEXR.InputFile(str(image_path)).header()['channels']
+    except OSError:
+        if image_path.exists():
+            with image_path.open('rb') as f:
+                sys.stderr.write(f"Broken file contents: {f.read(50)!r}\n")
+            sys.stderr.write(f"Broken file stats: {image_path.stat()}\n")
+        raise
+    if 'RenderLayer.Combined.R' in channels:
+        sys.exit("There is no support for OpenEXR multilayer")
```

Comment:
i think its better to raise here too, the shutdown should be controlled so it can clean up

Reply:
ok, done

Reply:
looks good!